fuzz_test.bzl

```diff
@@ -0,0 +1,35 @@
+#
+#  Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+""" This file will contain basic functions for fuzz test. """
+
+load("@rules_cc//cc:defs.bzl", "cc_test")
+
+def cc_fuzz_test(
+        name,
+        srcs,
+        linkopts = [],
+        deps = [],
+        visibility = None):
+    """ At present this cc_fuzz_test is just a wrapper of cc_test """
+
+    cc_test(
+        name = name + "_with_libfuzzer",
```

Comment:
Not sure if you need to adopt this "_with_libfuzzer" convention :)
I think when we build on top of this to optionally link unit test style on the corpus, it would be good to have the same test target name, which switches based on the --config (asan-fuzzer, or not)

Reply:
This sounds good to me. Let me strip the suffix!